modules/mod-deep-learning/ModelCard.cpp

```diff
@@ -0,0 +1,371 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+
+   ModelCard.cpp
+   Hugo Flores Garcia
+
+******************************************************************/
+
+#include "ModelCard.h"
+
+#include <wx/string.h>
+#include <wx/datetime.h>
+#include <wx/file.h>
+#include <wx/log.h>
+
+#include <rapidjson/document.h>
+#include <rapidjson/ostreamwrapper.h>
+#include <rapidjson/filewritestream.h>
+#include <rapidjson/writer.h>
+#include <rapidjson/filereadstream.h>
+#include <rapidjson/error/en.h>
+
+#include "CodeConversions.h"
+
+namespace validators 
+{
+
+   void validateExists(const std::string &key, const Doc& doc)
+   {
+      if(!doc.IsObject())
+         throw InvalidModelCardDocument(XO("The provided JSON document is not an object."), "");
```

Comment:
I have a mild preference for the new uniform-initialization, curly-brace syntax in constructor expressions.